src/annotator/config/test/index-test.js

```diff
@@ -191,16 +191,49 @@ describe('annotator.config.index', function() {
       assert.equal(configFrom(fakeWindow()).annotations, 'SOME_ANNOTATION_ID');
     });
 
-    it('ignores settings from JSON objects in the host page', function() {
-      fakeSharedSettings.jsonConfigsFrom.returns({foo: 'bar'});
+    [
+      'openLoginForm',
+      'openSidebar',
+    ].forEach(function(setting) {
+      it('still reads the ' + setting + ' setting from the JSON objects in the host page', function() {
+        var sharedSettings = {};
+        sharedSettings[setting] = 'bar';
+        fakeSharedSettings.jsonConfigsFrom.returns(sharedSettings);
+
+        assert.equal(configFrom(fakeWindow())[setting], 'bar');
+      });
 
-      assert.isUndefined(configFrom(fakeWindow()).foo);
+      it('still reads the ' + setting + ' setting from the hypothesisConfig() function in the host page', function() {
+        var configFuncSettings = {};
+        configFuncSettings[setting] = 'bar';
+        fakeConfigFuncSettingsFrom.returns(configFuncSettings);
+
+        assert.equal(configFrom(fakeWindow())[setting], 'bar');
+      });
     });
 
-    it('ignores settings from the hypothesisConfig() function in the host page', function() {
-      fakeConfigFuncSettingsFrom.returns({foo: 'bar'});
+    [
+      'showHighlights',
+      'branding',
+      'assetRoot',
+      'sidebarAppUrl',
```

Comment:
And yet the commit that introduced this change (no longer reading `sidebarAppUrl` from the host page) is already released, and the prod Chrome extension still seems to load `app.html` from the extension bundle. I'll have to look into this.

I think it's a separate issue though - _this_ PR just re-enables reading `openLoginForm` and `openSidebar` from the host page, which should fix the `/welcome` page issue. I seem to be unable to install the dev extension right now to test it though: https://hypothes-is.slack.com/archives/C1M8NH76X/p1497461454830189